packages/@n8n/api-types/src/dto/workflows/workflow-execution-status.dto.ts

```diff
@@ -0,0 +1,18 @@
+import { z } from 'zod';
+
+export const WorkflowExecutionStatusDtoSchema = z.object({
```

Comment:
I think for response types, we do not use the "DTO" terminology. We use the schema folder (at least that's what I've been doing :D)